data/default/ACCESSCONTROL-ROLES/roles.json

```diff
@@ -21,6 +21,21 @@
       "code": "SUPERUSER",
       "name": "Super User",
       "description": "Super User. Can change all master data and has access to all the system screens."
+    },
+    {
+      "code": "NATIONAL_SUPERVISOR",
```

Comment:
As discussed separately, having separate roles per boundary level is not in line with the required flexibility on platform. Please advise if product and platform team have agreed to this approach

Reply:
Have got confirmation on the above from Abhishek & Mrunal. Mrunal has also discussed this with Ganshyam and got confirmation on Jan, 11.

Reply:
@GhanshyamRawat-eGov please confirm that we need these additional boundary level specific roles.

Reply:
Hi @GhanshyamRawat-eGov, Could you please confirm on the above

Reply:
@egov-joy They have that requirment that dashboards are going to be boundary specific and that is why they are creating separate dashboards at each level.